### PR TITLE
main: add a way for the app to persist

### PR DESCRIPTION
**Pull request by cosimoc**
Make development easier.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -20,8 +20,12 @@ pkg.require({
     HackToolbox: '0',
 });
 
+const GLib = imports.gi.GLib;
```
Comment (ptomato) [inline src/main.js]: The new hotness is to use destructuring: `const {GLib} = imports.gi;`

File: src/main.js [hunk 2]
```diff
@@ -20,8 +20,12 @@ pkg.require({
...
 const {HackToolboxApplication} = imports.app;
 
 function main(argv) {
-    return (new HackToolboxApplication()).run(argv);
+    let app = new HackToolboxApplication();
+    if (GLib.getenv('HACK_TOOLBOX_PERSIST'))
+        app.hold();
```
Comment (ptomato) [inline src/main.js:29]: Seems fine as a quick hack even though we never release it. It's unfortunate the app class is hardcoded to have the IS_SERVICE flag set, but fixing that would be more trouble than this small PR warrants.

**Comment by ptomato** [comment]
All optional comments, feel free to merge.

File: src/main.js (hunk 2 above)
Comment (cosimoc) [reply src/main.js:29]: Yeah, that was too much yak shaving for something this simple. We use this pattern in other similar apps like the discovery feed.

File: src/main.js (hunk 1 above)
Comment (cosimoc) [reply src/main.js]: Fixed.

**Pull request merged**